WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -26,6 +27,8 @@
 
 import javax.inject.Inject;
 
+import de.greenrobot.event.EventBus;
```

Comment:
I'm not a fan of mixing event buses, but it was the simplest solution to communicate with the service.